resources/metadata/sfog.json

```diff
@@ -62,30 +64,278 @@
+          "All who are thirsty",
+          "All who are weak",
+          "Come to the fountain",
+          "Dip your heart in the stream of life",
+          "Let the pain and the sorrow",
+          "Be washed away",
+          "In the waves of His mercy",
+          "As deep cries out to deep"
         ],
         "c1": [
-          "",
-          "",
-          "",
-          ""
+          "We sing...",
+          "Come Lord Jesus come",
+          "Come Lord Jesus come"
+        ],
+        "b1": [
+          "Won't You come",
+          "Won't You come",
+          "Nothing but Your will for me",
+          "I am only free when You come"
+        ]
+      }
+    },
+    {
+      "title": "Hallelujah",
+      "songNumber": 4,
+      "author": "",
+      "music": "",
+      "capo": 0,
+      "lyrics": {
+        "v1": [
+          "An earthen vessel I was made",
+          "In me a treasure has been laid",
+          "In order that the glory would be of You.",
+          "In troubled times my soul resists",
+          "Until I find that gold exists",
+          "And finding this I sing my \"Hallelujahs\""
+        ],
+        "c1": [
+          "Hallelujah, Hallelujah",
+          "Hallelujah, Hallelu...jah"
+        ],
+        "v2": [
+          "My vessel is a fragile thing,",
+          "Seemed every day brought suffering",
+          "Until a part of me felt to refuse You.",
+          "I walled You out and turned away",
+          "And I resolved to never pray",
+          "But then I heard You whisper, \"Hallelujah\""
+        ],
+        "v3": [
+          "Of all the tears I've ever cried",
+          "None were so dear as those that pried",
+          "Away the walls I built up to refuse You",
+          "I turned to You and Your embrace",
+          "As I beheld Your lovely face",
+          "Still echoed in my heart Your \"Hallelujah\""
+        ],
+        "v4": [
+          "How could it be, You were so kind",
+          "To make me weak, so I would find",
+          "The glory of the face of Jesus through You?",
+          "I will be made eternally",
+          "To match this gold inside of me,",
+          "And this is why I sing my \"Hallelujahs\""
+        ]
+      }
+    },
+    {
+      "title": "Center",
+      "songNumber": 5,
+      "author": "",
+      "music": "",
+      "capo": 0,
+      "lyrics": {
+        "c1": [
+          "O Christ be the center of our lives",
+          "Be the place we fix our eyes",
+          "Be the center of our lives"
+        ],
+        "v1": [
+          "And You're the center of the universe",
+          "Everything was made in You Jesus",
+          "Breath of every living thing",
+          "Everyone was made for You",
+          "You hold everything together",
+          "You hold everything together"
+        ],
+        "v2": [
+          "We lift our eyes to heaven",
+          "And we wrap our lives around Your life",
+          "We life our eyes to heaven to You"
```

Comment:
Thoughts of how we can convey the (2x) things sometimes? Just add it into the last line maybe?